src/organisms/cards/PercentageCard/index.js

```diff
@@ -31,18 +32,12 @@ function PercentageCard( props ) {
 
       <div className={styles['row']}>
         <div className={styles['col']}>
-          <TextLockup
-            variant='A'
-            header={header1}
-            subheader={subheader1}
-          />
+          <H3 typography={4} weight='light'>{header1}</H3>
```

Comment:
@patrickkim One of the reasons we didn't explicitly use specific tags in the components is because this should really be an implementation choice depending on the other headers on the page and SEO needs. The user should be able to use the percentage card with any tags